fuel-vm/src/interpreter/internal.rs

```diff
@@ -229,7 +231,7 @@ mod tests {
         let maturity = 0;
         let height = 0;
 
-        let script = Opcode::RET(0x01).to_bytes().to_vec();
+        let script = <[u8; 4]>::from(op::ret(0x01)).to_vec();
```

Comment:
maybe we should make a helper fn for this common pattern?

e.g.
```rust
fn serialize_ops<I: IntoIterator<Item = Instruction>>(ops: I) -> Vec<u8> {
...
}
```

Reply:
Personally I like the explicitness of `to_bytes().to_vec()`, which both autocompletes from types and is still composable. Added the `to_bytes` back to help with this.